src/runtime-rs/crates/hypervisor/src/device/pci_path.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) 2019-2023 Alibaba Cloud
+// Copyright (c) 2019-2023 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Context, Result};
+use lazy_static::lazy_static;
+use regex::Regex;
+
+// get pci path from bdf, and the bdf will be start with "0000:00"
+lazy_static! {
+    static ref RE: Regex = Regex::new(r"^0000:00:([[:xdigit:]]{2}).[[:xdigit:]]$").unwrap();
```

Comment:
- Could you give this a clearer name, maybe `PCI_BDF_REGEX` or `BDF_REGEX`?
- Could you add an example to the comment showing the fully expanded BDF format.

Reply:
fixed